ingest-app/src/cmr/ingest/services/messages.clj

```diff
@@ -15,93 +15,93 @@
         coll-ref-humanized-fields (for [[field value] collection-ref-fields]
                                     (format "%s [%s]" (vc/humanize-field field) value))]
     (format "Collection with %s referenced in granule [%s] provider [%s] does not exist."
-            (string/join ", " coll-ref-humanized-fields)
-            granule-ur provider-id)))
+            (util/html-escape (string/join ", " coll-ref-humanized-fields))
+            (util/html-escape granule-ur) (util/html-escape provider-id))))
```

Comment:
IS this meant to be granule-url not granule-ur? (I know this isn't part of your PR)

Reply:
ur is probably right.
https://cmr.earthdata.nasa.gov/search/site/docs/search/api.html#g-granule-ur